src/ucp/core/ucp_ep.h

```diff
@@ -435,6 +435,7 @@ typedef struct {
 #if UCS_ENABLE_ASSERT
     size_t                   ka_count; /* Number of KA rounds done */
 #endif
+    uint64_t                 client_id; /* Client EP ID */
```

Comment:
save in wireup_ep
in https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_ep.c#L593